### ISSUE TITLE
use libgepub for epub backend

### ISSUE THREAD

**Issue by yetist** [mate-desktop/atril#345]
[libgepub](https://gitlab.gnome.org/gnome/libgepub) may be a good choice for writing ePub backend.

**Comment by raveit65** [comment]
Great idea,
personal i am not very happy about current state of epub backend in atril.
Because origin author (a former GSoC student) has left mate and in fact it is unmaintained.
So switching to a new maintained upstream sounds great for me.
@mate-desktop/core-team

**Comment by lukefromdc** [comment]
I don't work with epubs at all (unless testing Atril with a provided one), but I second this: ditching unmaintained code for a properly maintained external library.

**Comment by rcaridade145** [comment]
As a stop-gap measure could it be considered the choice to add a default ebook reader?

**Comment by monsta** [comment]
Which ebook reader?

**Comment by rcaridade145** [comment]
> Which ebook reader?

https://babluboy.github.io/bookworm/ Seems to be the most appreciated written in GTK.

**Comment by monsta** [comment]
Ah, you mean setting the default app for epubs? That's up to distro maintainers...

**Comment by rcaridade145** [comment]
> Ah, you mean setting the default app for epubs? That's up to distro maintainers...

Yes that was what i meant. I brought it up because work had to be done to enable a default calculator to be set on Mate default options. Thought this could also be the case.

**Comment by raveit65** [comment]
...wrong topic.
Please do not hijack this report!
Topic becomes unreadable....
You need to edit your local mimetype settings.
But please do not ask in this report how to do this.

**Comment by rcaridade145** [comment]
> ...wrong topic.
> Please do not hijack this report!
> Topic becomes unreadable....
> You need to edit your local mimetype settings.
> But please do not ask in this report how to do this.

I was not asking how to do it. Didn't meant to "hijack" this report. Sorry.
